pkg/util/test/util.go

```diff
@@ -288,6 +288,42 @@ func ValidateStorageCluster(
 
 	pxEndpoint = fmt.Sprintf("%s:%d", svc.Spec.ClusterIP, servicePort)
 
+	if err := validatePodsByName(cluster.Namespace, "portworx-pvc-controller", timeout, interval); err != nil {
+		return err
+	}
+
+	if cluster.Spec.Stork != nil && cluster.Spec.Stork.Enabled {
+		if err := validatePodsByName(cluster.Namespace, "stork", timeout, interval); err != nil {
+			return err
+		}
+		if err := validatePodsByName(cluster.Namespace, "stork-scheduler", timeout, interval); err != nil {
+			return err
+		}
+	}
+
+	if cluster.Spec.Autopilot != nil && cluster.Spec.Autopilot.Enabled {
+		if err := validatePodsByName(cluster.Namespace, "autopilot", timeout, interval); err != nil {
+			return err
+		}
+	}
+
+	if cluster.Spec.UserInterface != nil && cluster.Spec.UserInterface.Enabled {
```

Comment:
Should also validate image, see example from my PR portworx/spawn#518
`		// Lighthouse
		if spec.Spec.UserInterface != nil {
			if spec.Spec.UserInterface.Enabled {
				params.EnableLighthouse = "true"
				if spec.Spec.UserInterface.Image != "" {
					params.EnableLighthouse = spec.Spec.UserInterface.Image
				}
			}
		}`

Reply:
Done